modules/jaggery-apps/app-publisher-web/apis/v1/lifecycle_api_router.jag

```diff
@@ -21,553 +21,805 @@
  Created Date: 6/8/2013
  */
 
```

Comment:
Please fix this file as code review comments in https://github.com/wso2/product-app-manager/pull/234